### ISSUE TITLE
Automated merges from master into release branches should be assigned to the original submitter

### ISSUE THREAD

**Issue by phargogh** [natcap/invest#127]
When Dave submits a PR into `master` that is reviewed by Doug, once that PR is merged into `master`, our automation starts a new PR from `master` into each `release/*` branch and assigns the PR to Doug.

Instead, Dave really has the best context for the change going in and would be best suited to be responsible for making sure that change is PRed into `release/*` appropriately.  So, Dave should be assigned to review the release PR rather than Doug.

**Comment by phargogh** [comment]
Duplicates #119
